test/0.8.9/unstructured-storage.test.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.24 <0.9.0;
+
+import "forge-std/Test.sol";
+import { UnstructuredStorage } from "contracts/0.8.9/lib/UnstructuredStorage.sol";
+
+contract ExposedUnstructuredStorage {
```

Comment:
Please consider addressing the following warnings:

```bash
Warning (2018): Function state mutability can be restricted to view
 --> test/0.8.9/unstructured-storage.test.sol:8:5:
  |
8 |     function _getStorageBool(bytes32 position) public returns (bool) {
  |     ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to view
  --> test/0.8.9/unstructured-storage.test.sol:12:5:
   |
12 |     function _getStorageAddress(bytes32 position) public returns (address) {
   |     ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to view
  --> test/0.8.9/unstructured-storage.test.sol:16:5:
   |
16 |     function _getStorageBytes32(bytes32 position) public returns (bytes32) {
   |     ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to view
  --> test/0.8.9/unstructured-storage.test.sol:20:5:
   |
20 |     function _getStorageUint256(bytes32 position) public returns (uint256) {
   |     ^ (Relevant source part starts here and spans across multiple lines).
   ```

Reply:
Fixed.